communicator/winrm/provisioner.go

```diff
@@ -19,6 +19,9 @@ const (
 	// DefaultPort is used if there is no port given
 	DefaultPort = 5985
 
+	// DefaultSslPort is used if there is no port given and HTTPS is true
+	DefaultSslPort = 5986
```

Comment:
Let's call this something else, since it's not going to be using SSL. Maybe `DefaultHTTPSPort` which matches the config option.

Reply:
done